trident/absorption_spectrum/absorption_spectrum.py

```diff
@@ -402,8 +402,8 @@ def _add_continua_to_spectrum(self, field_data, use_peculiar_velocity,
         # continuum effect, we normalize min_tau by the n_absorbers.
         n_absorbers = field_data['dl'].size
 
-        if n_absorbers == 0:
-            mylog.info("No absorbers in path of LightRay.")
+        if n_absorbers == 0: 
```

Comment:
Would you mind going through this PR and removing the extra whitespace that you included (like here) that is adding to the diff associated with this PR?  There are a bunch of spots where you've done this.

Reply:
ok working on it